File: 1822-sign-of-the-product-of-an-array/1822-sign-of-the-product-of-an-array.cpp
class Solution {
public:
    int arraySign(vector<int>& nums) {
        int negs = 0;
        for (int &number : nums) {
            if (number == 0) return 0;
            if (number < 0) ++negs;
        }
        return negs % 2 ? -1 : 1;
    }
};

File: 1822-sign-of-the-product-of-an-array/1822-sign-of-the-product-of-an-array.py
class Solution:
    def arraySign(self, nums: List[int]) -> int:
        negs = 0
        for num in nums:
            if num < 0:
                negs += 1
            elif num == 0:
                return 0
        if negs % 2 == 1:
            return -1
        else:
            return 1